section3/README.md

```diff
@@ -33,10 +33,18 @@ One challenge learners in a new environment/content area face is feeling like th
 So this week, we would like you to start self-monitoring your progress. Before you start on the technical work, reflect back on how the project has gone so far.
 
 - Have the time estimates matched up with your experience?
+
+  * The time estimates that I had set up in my calendar for the project have matched up pretty well for each of the sections of my project. There was only one section 2 exercise where the timing went past what I had originally thought would be necessary. This was the section on making our own little ruby game, but once I had the correct indentation it went by much quicker. I would say 90% of the time estimates that I had made ended up working out perfectly.
```

Comment:
That's awesome! And your point on line 41 about sitting down with a clear goal is HUGE - probably related to your success with time management/monitoring.